sdc/Sdc.tcl

```diff
@@ -406,6 +406,36 @@ proc current_design { {design ""} } {
 
 ################################################################
 
+# Generic filter_objs command.
+proc filter_objs { filter objects filter_function filter_type error_code } {
+  set filter_regexp1 {@?([a-zA-Z_]+) *(==|!=|=~|!~) *([0-9a-zA-Z_\*]+)}
+  set filter_or_regexp "($filter_regexp1) *\\|\\| *($filter_regexp1)"
```

Comment:
filter_or_regexp and filter_and_regexp are referenced here but the definitions were removed